### ISSUE TITLE
FAQ out of date?

### ISSUE THREAD

**Issue by WhyNotHugo** [evansd/whitenoise#264]
> Shouldn’t I be pushing my static files to S3 using something like Django-Storages?
> No, you shouldn’t. The main problem with this approach is that Amazon S3 cannot currently selectively serve compressed content to your users. Compression (using either the venerable gzip or the more modern brotli algorithms) can make dramatic reductions in the bandwidth required for your CSS and JavaScript. But in order to do this correctly the server needs to examine the Accept-Encoding header of the request to determine which compression formats are supported, and return an appropriate Vary header so that intermediate caches know to do the same. This is exactly what WhiteNoise does, but Amazon S3 currently provides no means of doing this.

Looks like this is out of date. [CloudFront supports compressing files](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html), and, since September 2020, [supports Brotli compression](https://aws.amazon.com/about-aws/whats-new/2020/09/cloudfront-brotli-compression/).

**Comment by evansd** [comment]
Thanks

**Comment by gabn88** [comment]
@WhyNotHugo 
What configuration do you currently use to make it work with CloudFront automatically compressing files? Does CloudFront charge you extra when you add compression?

**Comment by WhyNotHugo** [comment]
> This is good news, I reverted back to the Django ManifestStatiFileStorage (instead of the compressed one) [...]

I missed this comment at the time (there's no email notification if a comment is edited). Thanks for pointing this out, I'm still using `CompressedManifestStaticFilesStorage`; that's so dumb of me :facepalm:

**Comment by Jsalaz1989** [comment]
Does this mean [this part](http://whitenoise.evans.io/en/stable/django.html#using-compression-algorithms-other-than-gzip) of the docs is also outdated? Do I still need to [enable brotli in whitenoise](http://whitenoise.evans.io/en/stable/django.html#brotli-compression) via `pip install whitenoise[brotli]` or is it all taken care of on the CloudFront side?

According to [Amazon](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html#compressed-content-custom-origin):

"CloudFront can compress objects using the Gzip and Brotli compression formats. When the viewer supports both formats, CloudFront prefers Brotli."

**Comment by WhyNotHugo** [comment]
> Does this mean this part of the docs is also outdated?

Yes, a PR for that would be welcome. Support for gzip and brotli needs to be enabled on cloudfront too, btw.
